recipes/ipypdf/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: ipypdf
+  version: {{ version }}
+
+source:
+  url: https://github.com/joelstansbury/ipypdf/archive/0206fb7.tar.gz
```

Comment:
Even if you _do_ stay with the github upstream, you'll need to use (sensible pattern of) tags, e.g. `v0.0.1`, and use that in the URL. This will help the bot create good PRs for you, which will help reduce your burden.